docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -47,7 +47,7 @@ yarn add @material-ui/core
 
 ### Divider
 
-- [Divider] Use border instead of background color. It prevents inconsistent height on scaled screens. For people customizing the color of the border, the change requires changing the override CSS property:
+- Use border instead of background color. It prevents inconsistent height on scaled screens. For people customizing the color of the border, the change requires changing the override CSS property:
```

Comment:
Looking at the migration guide of v4, the `[Divider]` was meant to tell users the exact module it impacts, for instance: https://material-ui.com/guides/migration-v3/#dialog. I guess it wasn't that helpful, we can have the same upside with a descriptive diff.

Reply:
I would put each module under a h3. Makes it simple to search for affected modules. Making an opinionated distinction for what gets a heading and what not is more confusing for changelogs.

Reply:
Sounds great too